### PR TITLE
swaymsg: add page

### PR DESCRIPTION
**Pull request by theryangeary**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by theryangeary** [comment]
I thought that might be a concern. I wasn't sure how to best approach that. It could be `swaymsg {{ipc_message}}` instead, but I was thinking it would be helpful to actually enumerate the 10 options (workspaces,inputs,outputs,tree,seats,marks,bar_config,version,binding_modes,config), because without that, users would just have to check the man page anyway. Thoughts?

**Comment by agnivade** [comment]
Select the top 2 options and go with it. Mention each of them separately.

File: pages/common/swaymsg.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+`swaymsg -t get_seats`
+
+- Send sway command - see sway(5) for list of commands:
+
+`swaymsg -- {{command}}`
+
+- Pretty print JSON data:
+
+`swaymsg --pretty -- {{command}}`
```
Comment (schneiderl) [inline pages/common/swaymsg.md]: Does this space between `--` and the `command` should really exist?

File: pages/common/swaymsg.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# swaymsg
+
+> Send messages to a running instance of sway over the IPC socket.
```
Comment (agnivade) [inline pages/common/swaymsg.md]: Add another subheading here - > See also: sway

File: pages/common/swaymsg.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Get outputs available to sway:
+
+`swaymsg -t get_outputs`
+
+- Get list of seats:
+
+`swaymsg -t get_seats`
+
+- Send sway command - see sway(5) for list of commands:
```
Comment (agnivade) [inline pages/common/swaymsg.md]: Send sway command (use `sway` to get the list of commands)

**Comment by sbrl** [comment]
Hey, @theryangeary! Note that there's no need to force-push - we have a button for that when merging :-)

Also, the GH web interface works really well for updating PRs :D

**Comment by zdroid** [comment]
@stale You must be quite impatient

**Comment by agnivade** [comment]
@zdroid - Unless you see something pending from our side, I'd suggest to leave it closed and then the PR author reopen it.

**Comment by zdroid** [comment]
Forgot to re-close it.

**Pull request closed without merging**